src/events.ts

```diff
@@ -29,6 +29,13 @@ export type DeviceLowBatteryEvent = CommonDeviceEvent<
     battery_level: number
   }
 >
+export type DeviceBatteryStatusChanged = CommonDeviceEvent<
+  "device.battery_status_changed",
+  {
+    battery_status: string
+    battery_level: number
```

Comment:
I wasn't sure about this, but do we want to include the updated `battery_level` with this event type? I thought it might be useful information so I included it
@rchodava @seveibar

Reply:
I think this would be ok to include, even if it's going to get out of date since we include it in the `low_battery` event, wdyt @seveibar ?

Reply:
yea probably ok IMO